src/cil.ml

```diff
@@ -6746,7 +6751,7 @@ let rec xform_switch_stmt s break_dest cont_dest = begin
        * label_break: ; // break_stmt
        *
        * The default case, if present, must be used only if *all*
-       * non-default cases fail [ISO/IEC 9899:1999, �6.8.4.2, �5]. As
+       * non-default cases fail [ISO/IEC 9899:1999, �6.8.4.2, �5]. As
```

Comment:
I'm not too sure what happened here...perhaps my text editor messed something up? I can switch those back to the old characters.